exp-models/addon/authenticators/jam-jwt.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+import BaseAuthenticator from 'ember-simple-auth/authenticators/base';
+
+import { jwt_decode } from 'ember-cli-jwt-decode';
+
+import moment from 'moment';
+
+const {
+    $, RSVP
+} = Ember;
+
+export default BaseAuthenticator.extend({
```

Comment:
@samanehsan can you please reformat this code so it matches with https://github.com/CenterForOpenScience/exp-addons/blob/develop/exp-models/.editorconfig and the other code in this repo?